.env.local

```diff
@@ -24,3 +24,28 @@ JS_REPORT_PACKAGE_PATH=
 #  !NOTE: Topics should comma seperated, optional include partion and repliction values
 #   e.g. <topic>:<partions>:<replicationFactor> -> test:3:2 (defaults to <topics>:3:1)
 # KAFKA_TOPICS=2xx,reprocess,3xx,metrics:3:1
+KAFKA_TOPICS=2xx,2xx-async,reprocess,3xx,metrics:3:3,patient,observation
+
+OPENHIM_CORE_MEDIATOR_HOSTNAME=openhimcomms.platform.jembi.cloud
+OPENHIM_MEDIATOR_API_PORT=443
+
+# Reverse Proxy - Nginx
+REVERSE_PROXY_INSTANCES=1
+DOMAIN_NAME=platform.jembi.cloud
+SUBDOMAINS=openhimcomms.platform.jembi.cloud,openhimcore.platform.jembi.cloud,openhimconsole.platform.jembi.cloud,kibana.platform.jembi.cloud,reports.platform.jembi.cloud,santewww.platform.jembi.cloud,santempi.platform.jembi.cloud,superset.platform.jembi.cloud,keycloak.platform.jembi.cloud,grafana.platform.jembi.cloud,minio.platform.jembi.cloud,jempi-web.platform.jembi.cloud,jempi-api.platform.jembi.cloud
+STAGING=false
+INSECURE=false
+
+# Identity Access Manager - Keycloak
+KC_FRONTEND_URL=https://keycloak.platform.jembi.cloud
+KC_GRAFANA_ROOT_URL=https://grafana.platform.jembi.cloud
+KC_JEMPI_ROOT_URL=https://jempi-web.platform.jembi.cloud
+KC_SUPERSET_ROOT_URL=https://superset.platform.jembi.cloud
+KC_OPENHIM_ROOT_URL=https://openhimconsole.platform.jembi.cloud
+GF_SERVER_DOMAIN=grafana.platform.jembi.cloud
+
+REACT_APP_JEMPI_BASE_API_HOST=https://jempi-api.platform.jembi.cloud
+REACT_APP_JEMPI_BASE_API_PORT=443
+OPENHIM_CONSOLE_BASE_URL=https://openhimconsole.platform.jembi.cloud
+OPENHIM_API_HOST=https://openhimcore.platform.jembi.cloud
+OPENHIM_API_PORT=443
```

Comment:
This is for remote deployments, and should not be in the local env file which is used for deploying locally. And also change the domain name to be `OPENHIM_API_HOST=https://openhimcore.<DOMAIN_NAME>`. Implementers can then sub in their domain name when deploying